packages/button/src/LionButton.js

```diff
@@ -87,6 +87,11 @@ export class LionButton extends DelegateMixin(SlotMixin(LionLitElement)) {
           fill: whitesmoke;
         }
 
+        :host(:active) .btn,
```

Comment:
If  understand right, this line will only apply to clicks on the host, the one beneath will apply to enter/click/space?

Reply:
This line applies to all clicks, the second line applies to Enter/Space.

Reply:
would be nicer to have all styles combined in one prop from Subclasser perspective (click interception). We can consider this once we would apply those modifiers on the host I think.